src/debugger/server/Debugger.js

```diff
@@ -299,18 +339,119 @@ export class DebugServer {
     return false;
   }
 
-  /*
-    Displays PP error message, then waits for user to run the program to
-    continue (similar to a breakpoint).
-  */
+  /**
+   * This function is only used if the original source locations in the
+   * sourcemaps are relative. This will discover the correct prefixes
+   * to use when converting the relative path to absolute paths.
+   */
+  _findSourcemapPrefixes(sourceMaps: Array<SourceFile> | void) {
```

Comment:
Since it is only called once during initialization, let's call it initializeSourcemapPrefixes().

Reply:
I originally decided against calling it "initialize" because it's run only when the sourcemaps are a certain format, whereas "initialize" implies to me that it would run unconditionally.